src/current/_includes/releases/v23.1/v23.1.8.md

```diff
@@ -0,0 +1,32 @@
+## v23.1.8
+
+Release Date: August 8, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-8-{{-site.data.products.enterprise-}}-edition-changes">{{ site.data.products.enterprise }} edition changes</h3>
+
+- [Azure Blob Storage credentials for implicit authentication](../v23.1/cloud-storage-authentication.html?filters=azure#azure-blob-storage-implicit-authentication) can now be stored in a file with the following YAML format:
+
+    ```
+    azure_tenant_id: MY_TENANT_ID,
+    azure_client_id: MY_CLIENT_ID,
+    azure_client_secret: MY_CLIENT_SECRET
+    ```
+    Set the environment variable `COCKROACH_AZURE_APPLICATION_CREDENTIALS_FILE` with the path of the file. The implicit authentication method now first looks for credentials in the file before proceeding to look at the rest of the chain specified in the DefaultAzureCredential class, such as environment variables and Managed Identity. These credentials can also be used for [Key Management Service (KMS)](../v23.1/take-and-restore-encrypted-backups.html#use-key-management-service). [#106688][#106688]
+
+<h3 id="v23-1-8-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where [`DROP INDEX ... CASCADE`](../v23.1/drop-index.html) could drop unrelated [foreign key constraints](../v23.1/foreign-key.html). For example, when `DROP INDEX ... CASCADE` was executed on a unique index for a table A, an unrelated foreign key constraint on a table B would be dropped if table B had a foreign key reference to table A. [#107793][#107793]
```

Comment:
Just tweaking the language a bit for clarity:

Fixed a bug where [`DROP INDEX ... CASCADE`](../v23.1/drop-index.html) could drop unrelated [foreign key constraints](../v23.1/foreign-key.html). For example, when `DROP INDEX ... CASCADE` was executed on a unique index for table A, unrelated outbound foreign key constraints on table B would be dropped, if table B had a foreign key reference to table A. [#107793][#107793] 

@rafiss @Xiang-Gu  Let me know if you have thoughts on the language here

Reply:
added "outbound"